static/src/javascripts/projects/common/modules/commercial/acquisitions-epic-testimonial-parameters.js

```diff
@@ -9,6 +9,10 @@ export type AcquisitionsEpicTestimonialTemplateParameters = {
     testimonialName: string,
 };
 
+export type AcquisitionsEpicTestimonialTemplateParametersGroup = Array<
```

Comment:
not sure I see the benefit it creating a type for this, especially as it's only used once. You can actually use `AcquisitionsEpicTestimonialTemplateParameters[]` as shorthand for `Array<AcquisitionsEpicTestimonialTemplateParameters>`, which makes it actually shorter than the name of the type alias